fastly/resource_fastly_service_dictionary_items_v1.go

```diff
@@ -0,0 +1,239 @@
+package fastly
+
+import (
+	"fmt"
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform/helper/schema"
+	"strings"
+)
+
+func resourceServiceDictionaryItemsV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceServiceDictionaryItemsV1Create,
+		Read:   resourceServiceDictionaryItemsV1Read,
+		Update: resourceServiceDictionaryItemsV1Update,
+		Delete: resourceServiceDictionaryItemsV1Delete,
+		Importer: &schema.ResourceImporter{
+			State: resourceServiceDictionaryItemsV1Import,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"service_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The service the dictionary belongs to",
+			},
+
+			"dictionary_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The dictionary the items belong to",
+			},
+
+			"items": {
+				Type:         schema.TypeMap,
+				Optional:     true,
+				Description:  "Key/value pairs that make up an item in the dictionary",
```

Comment:
I suggest prefixing this with "Map of ", ie:
"Map of key/value pairs that make up an item in the dictionary"

Reply:
Message was updated with new wording.